aws/kms/example/complete/outputs.tf

```diff
@@ -0,0 +1,15 @@
+output "key_id" {
+  value = module.kms.key_id
+}
+
+output "key_arn" {
+  value = module.kms.key_arn
+}
+
+output "key_alias" {
+  value = module.kms.key_alias
+}
+
+output "iam_policy" {
+  value = module.kms.iam_policy_kms_encrypt_decrypt_arn
+}
```

Comment:
should add description to these outputs, you can take reference from other modules